cryostatio/cryostat-web
# feat(clipboard): add clipboard copy to some helper text fields

Signed-off-by: Max Cao <macao@redhat.com>

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: https://github.com/cryostatio/cryostat-web/issues/695

## Description of the change:
This change adds allows a match expression example to be copied to the clipboard. As well as the Add Target modal example.

## Motivation for the change:
This change is helpful because users may want to copy the example for easier use.

## How to manually test:
1. Checkout pr and run the headless image with CORS.
2. Press add target button on dashboard and notice new change and try it.
3. Go to automated rules create form, and notice new change and try it.
